### PR TITLE
Add lint for Markdown

### PR DESCRIPTION
**Pull request by grushetsky**
This PR continues the story from #47.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,9 @@
   "main": "dist/index.js",
   "module": "dist/index-esm.js",
   "scripts": {
-    "lint": "eslint . && echo 'Lint finished.'",
+    "lint": "npm run lint-js && npm run lint-md && echo 'Lint finished.'",
```
Comment (ericelliott) [inline package.json]: When running npm scripts within npm scripts, it's handy to use `npm run -s` instead of `npm run` to reduce error output noise.

Comment (grushetsky) [reply package.json]: It is now [silent](https://github.com/ericelliott/autodux/pull/51/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R9). 🙂

**Comment by ericelliott** [comment]
Oops? Can you resolve conflicts and merge?

**Comment by grushetsky** [comment]
Sure! But let's merge #50 first, because it also introduces changes to _package.json_ and _package-lock.json_.

**Comment by ericelliott** [comment]
#50 merged.

**Comment by grushetsky** [comment]
@ericelliott, it is now ready to merge. 👌

**Comment by ericelliott** [comment]
Oops. Conflicts in package-lock.json. Probably my fault.

**Comment by grushetsky** [comment]
@ericelliott, conflicts resolved. 👌

**Pull request merged**